internal/slicer/slicer.go

```diff
@@ -86,6 +86,9 @@ func Run(options *RunOptions) error {
 		targetDir = filepath.Join(dir, targetDir)
 	}
 
+	if len(options.Archives) == 0 {
+		return fmt.Errorf("no valid archives defined")
+	}
```

Comment:
You get this: `error: cannot find package "test-package" in archive(s)`.

Reply:
I have been thinking about this and there is one concrete use-case I find problematic:
1. User attempts to configure an archive.
2. Chisel says that no valid archives are defined.
3. User fixes the archive definition.
4. Chisel still complains that it cannot find the package because, for example, the archive has negative priority.

Then, because of the change in error message between 2 and 4 the user thinks he has successfully configured the archive and that Chisel is not finding the package there, when in reality the archive configuration means Chisel won't look at it.

That is why I think we should drop this check and have a uniform error message ("cannot find package ..."), do you agree?

Reply:
Discusses offline and agreed to keep an uniform error message.

Reply:
Changed in db539fd6866b11e238444144efd9ec3dbaa544db.